homework02/sudoku.py

```diff
@@ -103,11 +114,13 @@ def find_possible_values(grid: tp.List[tp.List[str]], pos: tp.Tuple[int, int]) -
     >>> values == {'2', '5', '9'}
     True
     """
-    pass
+    num = [str(x) for x in range(1, 10)]
+    exn = [x for x in num if x not in get_block(grid, pos) + get_col(grid, pos) + get_row(grid, pos)]
+    return set(exn)
```

Comment:
`return {x for x in num if x not in get_block(grid, pos) + get_col(grid, pos) + get_row(grid, pos)}`